tracing-futures/examples/proxy_server.rs

```diff
@@ -152,7 +152,7 @@ impl AsyncRead for MyTcpStream {}
 
 impl AsyncWrite for MyTcpStream {
     fn shutdown(&mut self) -> Poll<(), io::Error> {
-        try!(self.0.lock().unwrap().shutdown(Shutdown::Write));
+        r#try!(self.0.lock().unwrap().shutdown(Shutdown::Write));
```

Comment:
Huh, weird.. Should this be using `?`

Reply:
probably --- this was a community contribution I think.